docs/src/modules/java/pages/views.adoc

```diff
@@ -42,18 +42,19 @@ As well as a Value Entity service `CustomerEntity.java` that will produce the st
 
 You implement a View by extending `kalix.javasdk.view.View` and subscribing to changes from an entity. You specify how to query it by providing a method annotated with `@Query`, which is then made accessible via REST annotations.
 
-.src/main/java/customer/view/CustomerByNameView.java
+.src/main/java/customer/view/CustomersByNameView.java
 [source,java]
 ----
-include::example$java-spring-customer-registry-views-quickstart/src/main/java/customer/view/CustomerByNameView.java[tags=class]
+include::example$java-spring-valueentity-customer-registry/src/main/java/customer/view/CustomersByNameView.java[tags=class]
```

Comment:
This was previously using a class from the quickstart, while all other snippets in this page comes from `java-spring-valueentity-customer-registry`. I don't know why we did like that. That was not the case for the protobuf docs. Somehow, we diverged here. 

Also, the callouts in the quickstart source code was in my way. 

To solve it, I created a new view `CustomersByNameView` in `java-spring-valueentity-customer-registry` and use it in this page.